src/audio/mixin_mixout/mixin_mixout.toml

```diff
@@ -21,22 +21,22 @@
 
 	REM # mod_cfg [PAR_0 PAR_1 PAR_2 PAR_3 IS_BYTES CPS IBS OBS MOD_FLAGS CPC OBLS]
 #if CONFIG_METEORLAKE
-	mod_cfg = [1, 0, 0, 0, 296, 4996000, 384, 384, 0, 4996, 0,
-			2, 0, 0, 0, 296, 2652000, 384, 384, 0, 2652, 0,
-			3, 0, 0, 0, 296, 2928000, 512, 512, 0, 2928, 0,
-			4, 0, 0, 0, 296, 2572000, 128, 128, 0, 2572, 0,
-			5, 0, 0, 0, 296, 3760000, 1536, 1536, 0, 3760, 0]
-#elif CONFIG_LUNARLAKE
+	mod_cfg = [ 1, 0, 0, 0, 296, 4996000, 384, 384, 0, 4996, 0,
+				2, 0, 0, 0, 296, 2652000, 384, 384, 0, 2652, 0,
+				3, 0, 0, 0, 296, 2928000, 512, 512, 0, 2928, 0,
+				4, 0, 0, 0, 296, 2572000, 128, 128, 0, 2572, 0,
+				5, 0, 0, 0, 296, 3760000, 1536, 1536, 0, 3760, 0]
+#elif defined(CONFIG_LUNARLAKE) || defined(CONFIG_PANTHERLAKE)
 	mod_cfg = [ 0, 0, 0, 0, 296, 644000, 45, 60, 0, 0, 0,
-			1, 0, 0, 0, 296, 669900, 48, 64, 0, 0, 0,
-			2, 0, 0, 0, 296, 934000, 96, 128, 0, 0, 0,
-			3, 0, 0, 0, 296, 1137000, 96, 128, 0, 0, 0,
-			4, 0, 0, 0, 296, 1482000, 48, 64, 0, 0, 0,
-			5, 0, 0, 0, 296, 1746000, 96, 128, 0, 0, 0,
-			6, 0, 0, 0, 296, 2274000, 192, 256, 0, 0, 0,
-			7, 0, 0, 0, 296, 2700000, 48, 64, 0, 0, 0,
-			8, 0, 0, 0, 296, 2964000, 96, 128, 0, 0, 0,
-			9, 0, 0, 0, 296, 3492000, 192, 256, 0, 0, 0]
+				1, 0, 0, 0, 296, 669900, 48, 64, 0, 0, 0,
+				2, 0, 0, 0, 296, 934000, 96, 128, 0, 0, 0,
+				3, 0, 0, 0, 296, 1137000, 96, 128, 0, 0, 0,
+				4, 0, 0, 0, 296, 1482000, 48, 64, 0, 0, 0,
+				5, 0, 0, 0, 296, 1746000, 96, 128, 0, 0, 0,
+				6, 0, 0, 0, 296, 2274000, 192, 256, 0, 0, 0,
+				7, 0, 0, 0, 296, 2700000, 48, 64, 0, 0, 0,
+				8, 0, 0, 0, 296, 2964000, 96, 128, 0, 0, 0,
+				9, 0, 0, 0, 296, 3492000, 192, 256, 0, 0, 0]
```

Comment:
Avoid white space changes next time to avoid breaking git rebase/merge/cherry-pick/revert etc.

https://wiki.openstack.org/wiki/GitCommitMessages